types/word2vector/word2vector-tests.ts

```diff
@@ -0,0 +1,37 @@
+import w2v = require("word2vector");
```

Comment:
Since index.d.ts uses ES6 exports, you should use the ES6 syntax `import * as w2v from "word2vector";` here